app/src/main/java/com/keylesspalace/tusky/entity/TrendingTagsResult.kt

```diff
@@ -0,0 +1,29 @@
+/* Copyright 2023 Tusky Contributors
+ *
+ * This file is a part of Tusky.
+ *
+ * This program is free software; you can redistribute it and/or modify it under the terms of the
+ * GNU General Public License as published by the Free Software Foundation; either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * Tusky is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even
+ * the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General
+ * Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with Tusky; if not,
+ * see <http://www.gnu.org/licenses>. */
+
+package com.keylesspalace.tusky.entity
+
+data class TrendingTag(
+    val name: String,
+    val url: String,
+    val history: List<TrendingTagHistory>,
+    val following: Boolean,
```

Comment:
Is this an expected property? It's not in the example at https://docs.joinmastodon.org/methods/trends/#tags, and the string "following" doesn't appear on that page.

Reply:
I believe so. It appears in this instance at least:

https://fosstodon.org/api/v1/trends/tags